### PR TITLE
Update log levels for the scheduler

### PR DESCRIPTION
**Pull request by foursixnine**
Current settings require openQA to be set to debug at least to be able
to see something in the logs, this commit sets the ground for which
messages to show in the log, as it provides useful information when
monitoring without running into major problems, or having too verbose
situation.

### CODE REVIEW

File: lib/OpenQA/Scheduler/Scheduler.pm [hunk 1]
```diff
@@ -217,13 +217,13 @@ sub schedule {
...
     if ($quit) {
-        log_debug("Exiting");
+        log_error("Exiting");
         exit(0);
     }
 
     my @allocated_jobs;
 
-    log_debug("-> Scheduling new jobs.");
+    log_info("-> Scheduling new jobs.");
```
Comment (Martchus) [inline lib/OpenQA/Scheduler/Scheduler.pm]: Isn't that one of the lines which are printed (annoyingly) often? If so, I wouldn't make it `log_info`.

Comment (foursixnine) [reply lib/OpenQA/Scheduler/Scheduler.pm]: no, this is one of the lines that are printed once per `schedule()` call

Comment (coolo) [reply lib/OpenQA/Scheduler/Scheduler.pm]: ... which happens annoyingly often in fullstack test

Comment (foursixnine) [reply lib/OpenQA/Scheduler/Scheduler.pm]: @coolo yeah, but it's good enough, the ammount of occurences is not as often as other messages. As a last resort we can log the full stack test's scheduler to a separate file, and cat it along the rest when there's a failure

**Comment by coolo** [comment]
don't merge this - feel free to rebase it on a reverted scheduler

**Comment by coolo** [comment]
rebase and send to revert_to_new_scheduler

**Comment by coolo** [comment]
I just found the github feature to change the target, so reopen

**Comment by coolo** [comment]
So just rebase :)

File: lib/OpenQA/Scheduler/Scheduler.pm [hunk 2]
```diff
@@ -59,13 +59,13 @@ CORE::state $no_actions = 0;
...
 sub normal_signals_handler {
-    log_debug("Received signal to stop");
+    log_info("Received signal to stop");
     $quit++;
     _reschedule(1, 1);
 }
 
 sub wakeup_scheduler {
-    log_debug("I've been summoned by the webui");
+    log_info("I've been summoned by the webui");
```
Comment (coolo) [inline lib/OpenQA/Scheduler/Scheduler.pm]: this happens too often to be info

File: lib/OpenQA/Scheduler/Scheduler.pm [hunk 3]
```diff
@@ -217,13 +217,13 @@ sub schedule {
 
     # Exit only when database state is consistent.
     if ($quit) {
-        log_debug("Exiting");
+        log_error("Exiting");
```
Comment (coolo) [inline lib/OpenQA/Scheduler/Scheduler.pm]: it's not an error - it just quits on external signals

**Comment by coolo** [comment]
Please reopen if still important to you (after rebase obviously :)

**Pull request closed without merging**